docs/AdviceForOssBeginners.md

```diff
@@ -0,0 +1,36 @@
+# Advice for OSS Beginners
+
+## Pay attention to instructions
+
+Take your time to learn and follow the workflow to the letter. 
+
+* If the setup instructions mention a specific version of a software to use, **use that exact version** 
+  instead of the latest version.
+* Do not skip any steps because you think that step is "not important". 
+* Do not assume current project does things the same way things were one in that _other_ project you are familiar with.
+
+## Follow others
+
+> OSS projects involve many persons contributing in small measures. If each person did things their own way, the project
+> will deteriorate into a mess very quickly. That is why being consistent is very important in OSS projects.
```

Comment:
I don't think this part needs to be block-quoted. It's good as normal paragraph.